experimental/dds/tree2/src/feature-libraries/sequence-field/sequenceFieldChangeEncoder.ts

```diff
@@ -20,110 +20,65 @@ function makeV0Codec<TNodeChange>(
 		encode: (changeset) => {
 			const jsonMarks: JsonCompatible[] = [];
 			for (const mark of changeset) {
+				// eslint-disable-next-line @typescript-eslint/consistent-type-assertions
```

Comment:
Can you explain the lint issue?

Reply:
The linter want me to write `const encodedMark: Mark<JsonCompatibleReadOnly> = { ...mark };` but that doesn't compile.